docs/branches.md

```diff
@@ -41,6 +40,59 @@ branch `main`. If the local target had also moved compared to `main@origin`
 merged. If one is ahead of the other, then that target will be the new target.
 Otherwise, the local branch will be conflicted (see next section for details).
 
+As of December 2023 Jujutsu tracks[^1] and fetches all branches by default, 
+which is confusing users coming from Git. To smoothen the transition branch 
+tracking was introduced. 
+
+### What does `git.auto-local-branches` actually do? 
+
+Jujutsu fetches all Git Refs under `origin/refs/branches` by default and then 
```

Comment:
`refs/remotes/origin` or `refs/remotes/<remote name>`

Reply:
We could say something like:

> By default, `jj git fetch --remote origin` does a few things: it 1) fetches all the branches from the remote `origin`, 2) stores them as remote-tracking branches, and 3) creates (or moves) the corresponding local branches. This is similar...

Reply:
I partially took ilya's phrasing here but dropped first part.